src/GitHub.UI/Converters/DurationToStringConverter.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Globalization;
+
+namespace GitHub.UI
+{
+    public class DurationToStringConverter : ValueConverterMarkupExtension<DurationToStringConverter>
+    {
+        public override object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            const int year = 356;
```

Comment:
Did you mean 365? I assume it just needs to be close so we can ignore leap years. :smile: